spec/requests/admin/administrators/success_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Admin::Administrators::Administrators", type: :request do
+  let(:name) { Faker::Name.name }
+  let(:email) { Faker::Internet.email }
+
+  before do
+    user = create(:user, :admin)
```

Comment:
you could move that into let(user){ ... }

Reply:
I think you're in the minority on this one: https://github.com/DFE-Digital/early-careers-framework/pull/114#discussion_r589297482

Reply:
👍